requirements.txt

```diff
@@ -1,2 +1,4 @@
--r core/requirements.txt
+# All the requirements should be listed in requirements-base.txt except
+# for psycopg2, which is listed here.
 -r requirements-base.txt
+psycopg2==2.8.6
```

Comment:
I think I've put this in the right place after combining requirements. Same for `psycopg2-binary`.